devito/passes/clusters/aliases.py

```diff
@@ -16,24 +16,24 @@
 
 
 @cluster_pass
-def cire(cluster, template, mode, options, platform):
+def cire(cluster, mode, sregistry, options, platform):
     """
     Cross-iteration redundancies elimination.
 
     Parameters
     ----------
     cluster : Cluster
         Input Cluster, subject of the optimization pass.
-    template : callable
-        To build the symbols (temporaries) storing the redundant expressions.
     mode : str
         The transformation mode. Accepted: ['invariants', 'sops'].
         * 'invariants' is for sub-expressions that are invariant w.r.t. one or
           more Dimensions.
         * 'sops' stands for sums-of-products, that is redundancies are searched
           across all expressions in sum-of-product form.
+    sregistry : SymbolRegistry
```

Comment:
Rephrase? Like "Devito symbol registery for temporaries. Creates unique temporary names...."

Reply:
done, updated all docstrings concerning `sregistry`